infrastructure/dogfood/terraform/aws-tf-module/main.tf

```diff
@@ -263,6 +263,7 @@ module "migrations" {
   ecs_service              = module.main.byo-vpc.byo-db.byo-ecs.service.name
   desired_count            = module.main.byo-vpc.byo-db.byo-ecs.appautoscaling_target.min_capacity
   min_capacity             = module.main.byo-vpc.byo-db.byo-ecs.appautoscaling_target.min_capacity
+  vuln_service             = module.vuln-processing.vuln_service_arn
```

Comment:
I think this might need to check for null, and use empty string (default) if null, otherwise use the value. Thinking of the scenario where vuln service has never existed or doesn't exist yet. Thoughts @rfairburn ?

Reply:
I think we made the default an empty string in the module, so worst case would be a first apply with it blank, but we can try it on the next new environment we spin up.